lib/pubsubstub/action.rb

```diff
@@ -12,7 +12,6 @@ class Action < Sinatra::Base
 
     def initialize(*)
       @channels = Hash.new { |h, k| h[k] = Channel.new(k) }
```

Comment:
If you change `PublishAction` to use `PubSubStub.publish` instead, you can get rid of this